mimesis/providers/person.py

```diff
@@ -36,6 +36,9 @@ def __init__(self, *args, **kwargs) -> None:
         self._store = {
             'age': 0,
         }
+        self._previously_generated: Dict[str, Set] = {
```

Comment:
Why do not use `self._store`? I guess `self._store` has been created for such things when you need to know previously generated values. What do you think? It's important for us to know your point, so feel free to oppose.

Reply:
I originally used ```self._store```. However, adding a new key value pair with type ```[str, Set]``` would result in a dictionary with multiple types of key value pairs. I wasn't sure if this was the cleanest way to do this.

Also note, when I implement ```Person.username(unique=True)``` and ```Person.telephone(unique=True)``` I will also be adding more ```[str, Set]``` key value pairs. Overall, I thought it would be cleaner to have a dedicated single type dictionary. It would only be used for keeping track of previously generated values for all the methods that implement ```unique=True```.

However, if you think I should just use ```self._store``` I will happily revise the code.

Reply:
Great!